lib/mpi/service.rb

```diff
@@ -165,12 +167,14 @@ def create_add_message(user_identity)
       MPI::Messages::AddPersonMessage.new(user_identity).to_xml if user_identity.icn_with_aaid.present?
     end
 
-    def create_profile_message(user_identity, search_type: MPI::Constants::CORRELATION_WITH_RELATIONSHIP_DATA)
+    def create_profile_message(user_identity,
+                               search_type: MPI::Constants::CORRELATION_WITH_RELATIONSHIP_DATA,
+                               orch_search: false)
       return message_icn(user_identity, search_type) if user_identity.mhv_icn.present?
       return message_edipi(user_identity, search_type) if user_identity.edipi.present? && Settings.mvi.edipi_search
       raise Common::Exceptions::ValidationErrors, user_identity unless user_identity.valid?
 
-      message_user_attributes(user_identity, search_type)
+      message_user_attributes(user_identity, search_type, orch_search: orch_search)
```

Comment:
I don't think this code will be reachable with orch_search, it'll fall into the `message_edipi` bucket instead, so I don't think `orch_search` will actually be called as expected